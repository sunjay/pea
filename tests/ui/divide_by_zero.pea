fn main() {
    println!(3 / 1);
    println!(3 / 0);
}
