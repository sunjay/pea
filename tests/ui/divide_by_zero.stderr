stack backtrace:
   0: main
error: attempt to divide by zero
